src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -401,6 +420,8 @@ APPLICATION_INFO::FindNativeAssemblyFromGlobalLocation(STRU* struFilename)
 //
 HRESULT
 APPLICATION_INFO::FindNativeAssemblyFromHostfxr(
+    HOSTFXR_OPTIONS* hostfxrOptions,
```

Comment:
HOSTFXR_OPTIONS can be replaced by MULTISZ

Reply:
How exactly? I need all 3 fields there, how can I replace all of them with a string wrapper?